src/receivers/verify-request.ts

```diff
@@ -44,7 +44,7 @@ export function verifySlackRequest(options: SlackRequestVerificationOptions): vo
 
   // Rule 1: Check staleness
   if (requestTimestampSec < fiveMinutesAgoSec) {
-    throw new Error(`${verifyErrorPrefix}: stale`);
+    throw new Error(`${verifyErrorPrefix}: x-slack-request-timestamp must differ from system time by no more than 5 minutes or request is stale`);
```

Comment:
Can you replace the hard-coded "5" in the log message? Perhaps, we can add a new local variable for the minutes above in this method and embed the value in the log.